ccloader/js/filemanager.js

```diff
@@ -1,7 +1,7 @@
 import { Db } from './db.js';
 
 const fs = require('fs');
-const path = require('path');
+const {posix: path} = require('path');

```

Comment:
Why should it be changed to explicitly take posix?

Reply:
Because chrome paths use /. posix handles this well.